pkg/lib/bundlev2/generate.go

```diff
@@ -0,0 +1,54 @@
+package bundlev2
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+// GenerateFunc generates a CSV for the bundle based on the resource within the bundle directory.
```

Comment:
nit: I have not seen this commentary style in Go code before - generally comments are full sentences for auto-formatting purposes (godoc). See https://golang.org/doc/effective_go.html#commentary